src/runner/index.js

```diff
@@ -740,12 +749,12 @@ export default class Runner extends EventEmitter {
     screenshots (...options) {
         let fullPage;
         let thumbnails;
-        let [path, takeOnFails, pathPattern] = options;
+        let [path, takeOnFails, pathPattern, pathPatternOnFails] = options;
```

Comment:
This way of assigning screenshot options is [deprecated](https://testcafe.io/documentation/402654/reference/testcafe-api/runner/screenshots). Please, leave it as is.